Content.Server/Medical/SuitSensors/SuitSensorComponent.cs

```diff
@@ -87,3 +87,7 @@ public sealed partial class SuitSensorComponent : Component
     [DataField, ViewVariables]
     public bool PreviousControlsLocked = false;
 }
+
+public sealed class SuitSensorsSendAttemptEvent : CancellableEntityEventArgs
+{
+};
```

Comment:
Make it record struct raised by ref and just add a bool field to it plz.

Reply:
I just reverted doing that because I thought that was what I was supossed to do :despair:. One minute.